AppCenterDataStorage/AppCenterDataStorage/MSSerialization.h

```diff
@@ -0,0 +1,27 @@
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@protocol MSSerialization
```

Comment:
MSDocumentSerialization since there is global namespace pollution otherwise I think.